runtime-dev-innerloop (Build windows x86 release MSBuild_CMake)  is

```
dotnet-install: To check the list of dependencies, go to https://docs.microsoft.com/dotnet/core/install/windows#dependencies
dotnet-install: Installation finished
D:\workspace\_work\1\s\.dotnet\sdk\6.0.100-preview.4.21255.9\MSBuild.dll /nologo -maxcpucount /m -verbosity:m /v:minimal /bl:D:\workspace\_work\1\s\artifacts\log\Release\ToolsetRestore.binlog /clp:Summary /clp:ErrorsOnly;NoSummary /nr:False /p:ContinuousIntegrationBuild=True /p:TreatWarningsAsErrors=true /p:__ToolsetLocationOutputFile=D:\workspace\_work\1\s\artifacts\toolset\6.0.0-beta.21359.3.txt /t:__WriteToolsetLocation /warnaserror D:\workspace\_work\1\s\artifacts\toolset\restore.proj
D:\workspace\_work\1\s\artifacts\toolset\restore.proj : error : Unable to load the service index for source https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-eng/nuget/v3/index.json.
D:\workspace\_work\1\s\artifacts\toolset\restore.proj : error : Unable to load the service index for source https://pkgs.dev.azure.com/dnceng/public/_packaging/nuget-build/nuget/v3/index.json.
D:\workspace\_work\1\s\artifacts\toolset\restore.proj : error : Unable to find package Microsoft.DotNet.Arcade.Sdk with version (= 6.0.0-beta.21359.3)
D:\workspace\_work\1\s\artifacts\toolset\restore.proj : error :   - Found 178 version(s) in dotnet-tools [ Nearest version: 5.0.0-beta.19558.13 ]
D:\workspace\_work\1\s\artifacts\toolset\restore.proj : error :   - Found 0 version(s) in richnav
D:\workspace\_work\1\s\artifacts\toolset\restore.proj : error :   - Found 0 version(s) in dotnet-eng
D:\workspace\_work\1\s\artifacts\toolset\restore.proj : error :   - Found 0 version(s) in dotnet-public
D:\workspace\_work\1\s\artifacts\toolset\restore.proj : error :   - Found 0 version(s) in dotnet6-transport
D:\workspace\_work\1\s\artifacts\toolset\restore.proj : error :   - Found 0 version(s) in dotnet6
D:\workspace\_work\1\s\artifacts\toolset\restore.proj : error :   - Found 0 version(s) in nuget-build
D:\workspace\_work\1\s\artifacts\toolset\restore.proj : error MSB4236: The SDK 'Microsoft.DotNet.Arcade.Sdk' specified could not be found.
Build failed with exit code 1. Check errors above.
See log: D:\workspace\_work\1\s\artifacts\log\Release\ToolsetRestore.binlog
```

--- a/src/libraries/sendtohelixhelp.proj
+++ b/src/libraries/sendtohelixhelp.proj
@@ -317,8 +317,8 @@
       <WasmBuildTargetsDir>$([MSBuild]::NormalizeDirectory('$(RepoRoot)', 'src', 'mono', 'wasm', 'build'))</WasmBuildTargetsDir>
       <TestEchoMiddleware>$([MSBuild]::NormalizeDirectory('$(ArtifactsDir)', 'bin', 'NetCoreServer', '$(NetCoreAppCurrent)-$(Configuration)'))</TestEchoMiddleware>
       <RemoteLoopMiddleware>$([MSBuild]::NormalizeDirectory('$(ArtifactsDir)', 'bin', 'RemoteLoopServer', '$(NetCoreAppCurrent)-$(Configuration)'))</RemoteLoopMiddleware>
-      <WorkItemPrefix Condition="'$(Scenario)' == 'BuildWasmApps' and '$(TestUsingWorkloads)' == 'true'">Workloads/</WorkItemPrefix>
-      <WorkItemPrefix Condition="'$(Scenario)' == 'BuildWasmApps' and '$(TestUsingWorkloads)' != 'true'">EMSDK/</WorkItemPrefix>
+      <WorkItemPrefix Condition="'$(Scenario)' == 'BuildWasmApps' and '$(TestUsingWorkloads)' == 'true'">Workloads-</WorkItemPrefix>
+      <WorkItemPrefix Condition="'$(Scenario)' == 'BuildWasmApps' and '$(TestUsingWorkloads)' != 'true'">EMSDK-</WorkItemPrefix>
     </PropertyGroup>
 
     <PropertyGroup Condition="'$(RuntimeFlavor)' == 'Mono'">

